programs/create-genesis/ee_genesis/genesis_ee_builder.hpp

```diff
@@ -47,6 +45,7 @@ class genesis_ee_builder final {
     void build_pinblocks();
     void build_accounts();
     void build_funds();
+    void write_genesis_converts();
```

Comment:
`build_genesis_converts()`
Or remove all another methods. It is preferred. It is small and "bug-safe" change.

Reply:
fixed in 8bc714b, ec6accd